src/figuresClasses.ts

```diff
@@ -1,11 +1,100 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle';
+
+  color: 'red' | 'green' | 'blue';
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
+    a: number,
+    b: number,
+    c: number,
+  ) {
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error('Invalid value. It should be a positive number.');
+    }
+
+    if (a >= b + c || b >= a + c || c >= a + b) {
+      throw new Error(
+        'The longest side of triangle should be bigger than sum of others',
```

Comment:
[CODE STYLE] - The error message here is good, but it could be slightly more descriptive. Consider rewording to 'The sum of any two sides of a triangle must be greater than the third side.'